api/servers_handler_test.go

```diff
@@ -0,0 +1,70 @@
+package api_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/topfreegames/pitaya-admin/api"
+	pt "github.com/topfreegames/pitaya-admin/testing"
+)
+
+func TestGetServerList(t *testing.T) {
+	t.Parallel()
+	if pt.IsConf == false {
+		pt.ConfApp()
+	}
+	a := pt.TestApp
+	var (
+		response      *httptest.ResponseRecorder
+		serverHandler = api.NewServersHandler(a)
+	)
+
+	tables := map[string]struct {
+		request *http.Request
+		asserts func(response *httptest.ResponseRecorder)
+	}{
+		"test_get_servers": {
```

Comment:
We don't need to test if all servers are returned here, since pitaya's GetServers function should already have tested it, but we'd like to test if our API is returning something coherent. For example, if I change the handler to call `pitaya.GetServers()` and then ignore the output and return some random stuff this test would still pass.